src/locales/en_US.ts

```diff
@@ -1405,7 +1405,7 @@ const strings = {
   bank_info_title: 'Bank Info',
   home_address_title: 'Home Address',
   input_output_currency: 'Currency',
-  n_a: 'n/a',
+  n_a: 'N/A',
```

Comment:
Instead of "n_a", could it be "empty_notes_placeholder" or something like that? Other languages might want to express "empty notes" using other idioms.

Reply:
I'll add another placeholder for notes, but I want to change 'N/A' regardless because it's being used elsewhere